kyo-logging-jpl/jvm/src/main/scala/kyo/logging/LogPlatformSpecific.scala

```diff
@@ -0,0 +1,59 @@
+package kyo.logging
+
+import System.Logger.Level
+import kyo.AllowUnsafe
+import kyo.Frame
+import kyo.Log
+import kyo.Text
+
+object LogPlatformSpecific:
+
+    /** WARNING: Low-level API meant for integrations, libraries, and performance-sensitive code. See AllowUnsafe for more details. */
+    object Unsafe:
+
+        object JPL:
```

Comment:
I think the `JPL` indirection isn't very valuable. We could have `class Unsafe` and `object Unsafe` directly under `LogPlatformSpecific` (or its new name if we change it)

Reply:
I agree, I will re-organize the constructors as suggested by @johnhungerford